prototype_source/fx_graph_mode_ptq_static_tutorial.py

```diff
@@ -0,0 +1,446 @@
+"""
+(prototype) FX Graph Mode Post Training Static Quantization
+===========================================================
+
+**Author**: `Jerry Zhang <https://github.com/jerryzh168>`_
+
+This tutorial introduces the steps to do post training static quantization in graph mode based on
+`torch.fx <https://github.com/pytorch/pytorch/blob/master/torch/fx/__init__.py>`_. 
+The advantage of FX graph mode quantization is that as long as the model can be symbolically traced, 
+we can perform quantization fully automatically on the model. 
+
+We have a separate tutorial for FX Graph Mode Post Training Dynamic Quantization(todo).	
```

Comment:
Will this todo be replaced by a link before this PR is merged?

Reply:
this will be added later, I'll have another PR to add tutorial for dynamic quantization